packages/template-retail-react-app/app/components/display-price/index.jsx

```diff
@@ -7,61 +7,84 @@
 
 import React from 'react'
 import PropTypes from 'prop-types'
-import {Skeleton, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {Box, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
 import {useIntl} from 'react-intl'
 import {useCurrency} from '@salesforce/retail-react-app/app/hooks'
 
 const DisplayPrice = ({
-    basePrice,
-    discountPrice,
-    isProductASet = false,
+    strikethroughPrice,
+    currentPrice,
+    prefixLabel,
     currency,
-    discountPriceProps,
-    basePriceProps,
-    skeletonProps
+    currentPriceProps,
+    strikethroughPriceProps
 }) => {
     const intl = useIntl()
     const {currency: activeCurrency} = useCurrency()
+    const currentPriceText = intl.formatNumber(currentPrice, {
```

Comment:
Question: Why do we use the `currency` props in some places and not in others? e.g. is looks like we don't use it when using the DisplayPrice component in the ProductTile, but we do use it when using the DisplayPrice component in the add to cart modal hook and product-view?

Reply:
Good point. I don't recall the exact reason, but my guess was `currency` prop in ProductView can be `undefined` since we are passing `product?.currency`. For consistency, I'd opt to have DisplayPrice take currency as a prop and remove the useCurrency hook. See [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1760/files#diff-0a507e903ea074f28e2f5e8c3f0613a7dd7c13f847c4ab27f8ceeac87ab00d10R309)
```
// current implementation
<ProductView
   currency={product?.currency}



// fix
const {currency: activeCurrency} = useCurrency()
<ProductView
   currency={product?.currency || activeCurrency}
```